providers/aws/services/accessanalyzer/accessanalyzer_enabled_without_findings/accessanalyzer_enabled_without_findings.py

```diff
@@ -0,0 +1,39 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.accessanalyzer.accessanalyzer_client import (
+    accessanalyzer_client,
+)
```

Comment:
You have to import the logger and put all the check code inside a `try-except` clause. We can review it later.

Take a look here for guidance: https://github.com/prowler-cloud/prowler/pull/1354/files#diff-f4a5355acbd446b447c264f969454e9716b627cef9920a3cf6af42cffca451b7